#!/usr/bin/env python
PACKAGE = "pal_vision_segmentation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("threshold", int_t, 0, "Threshold", 89, 0, 255)
gen.add("dilate_iterations", int_t, 0, "Dilate iterations", 7, 0, 20)
gen.add("dilate_size", int_t, 0, "Dilate kernel size", 5, 3, 15)
gen.add("erode_iterations", int_t, 0, "Erode iterations", 1, 0, 20)
gen.add("erode_size", int_t, 0, "Erode kernel size", 3, 3, 15)

exit(gen.generate(PACKAGE, "disparity_segmentation_node", "DisparitySegment"))
